code-challenges.js

```diff
@@ -20,25 +20,58 @@ const fruit4 = "kiwi"
 
 // Pseudo code:
 
-
 // --------------------1) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
-// Pseudo code:
+// Pseudo code: 
+//  Combine the 2 provided variables
+//  Get the the length of the new array
+//  Return Expected output of 9
+
+// Code:
+var comboRun = (padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns))
+console.log(comboRun.length)
+//  Output ----> 9
```

Comment:
This looks good. My only refactor consideration would be doing this: `var comboRun = padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns).length`